legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -178,10 +178,10 @@ class FilingTitles(str, Enum):
         'title': 'Consent Continuation Out',
         'displayName': '6-Month Consent to Continue Out',
         'codes': {
-            'BC': 'CONTO',
-            'BEN': 'CONTO',
-            'ULC': 'CONTO',
-            'CC': 'CONTO'
+            'BC': 'COUTI',
+            'BEN': 'COUTI',
+            'ULC': 'COUTI',
+            'CC': 'COUTI'
```

Comment:
The intent of #16254 is to add fee code entries for the new "Continuation Out" filing.  Please add a separate entry instead of updating the `consentContinuationOut` fee codes.  This will of course apply to the rest of the code changes you have made.